ravenframework/Models/PostProcessors/Validations/PPDSS.py

```diff
@@ -114,6 +136,18 @@ def _handleInput(self, paramInput):
         self.pivotParameterFeature = child.value
       elif child.getName() == 'pivotParameterTarget':
         self.pivotParameterTarget = child.value
+      elif child.getName() == 'separateFeatureData':
+        if 'type' not in child.parameterValues.keys():
+          self.raiseAnError(IOError, 'Tag separateFeatureData must have attribute "type"')
+        else:
+          self.separateFeatureType = child.parameterValues["type"]
+          self.separateFeatureData = child.value
+      elif child.getName() == 'separateTargetData':
+        if 'type' not in child.parameterValues.keys():
+          self.raiseAnError(IOError, 'Tag separateTargetData must have attribute "type"')
```

Comment:
see above comment

Reply:
Have removed the error message and now use the default.